### PR TITLE
fr_FR update

### PR DESCRIPTION
**Pull request by Machou**

### CODE REVIEW

File: client/strings/fr.json [hunk 1]
```diff
@@ -39,8 +39,8 @@
   "ButtonMatchBooks": "Rechercher les Livres",
   "ButtonNevermind": "Oubliez cela",
```
Comment (Machou) [inline client/strings/fr.json:40]: can i get context please ?

Comment (advplyr) [reply client/strings/fr.json:40]: Restoring backups

image

**Comment by Machou** [comment]
Is good. You can valide:)

**Comment by advplyr** [comment]
Thanks~!

**Pull request merged**